cluster/gce/util.sh

```diff
@@ -1048,6 +1058,7 @@ function prepare-push() {
     # being used, create a temp one, then delete the old one and recreate it once again.
     create-node-instance-template "tmp"
```

Comment:
You might have collisions if you have multiple clusters in the same project.  I'd make this "${NODE_INSTANCE_PREFIX}-template-tmp" as it was, effectively, before.